elao.app.docker/.manala/github/deliveries/setup/action.yaml.tmpl

```diff
@@ -0,0 +1,76 @@
+name: Set up
+
+description: Deliveries - Set up
+
+inputs:
+  app:
+    description: App
+    required: false
+  tier:
+    description: Tier
+    required: true
+  ref:
+    description: Ref
+    required: false
+    default: main
+  env:
+    description: Env vars as a single string
+    required: false
+  services:
+    description: Services
+    required: false
+#  ssh_key:
+#    description: Ssh Key
+#    required: false
+  secrets:
+    description: Secrets
+    required: true
```

Comment:
> **Note**: I reintroduce the secrets here rather than expecting the `ssh_key` directly, since we currently have a config entry in `.manala.yaml` deliveries that allows to autoselect this (same for the deploy URL). So let's move on by keeping this for existing projects for now, and determinate a better way later if we decide to drop these keys.